spec/models/user_spec.rb

```diff
@@ -1,34 +1,34 @@
-require 'rails_helper'
+# require 'rails_helper'
 
-RSpec.describe User, type: :model do
-  describe 'User model' do
-    subject { User.new(name: 'Amy', photo: 'Tom.png', bio: 'bio', posts_counter: 0) }
-    before { subject.save }
+# RSpec.describe User, type: :model do
+#   describe 'User model' do
+#     subject { User.new(name: 'Amy', photo: 'Tom.png', bio: 'bio', posts_counter: 0) }
+#     before { subject.save }
 
-    it 'Check the name is not blank' do
-      subject.name = nil
-      expect(subject).to_not be_valid
-    end
+#     it 'Check the name is not blank' do
+#       subject.name = nil
+#       expect(subject).to_not be_valid
+#     end
 
-    it 'check if the posts_counter numeric' do
-      subject.posts_counter = ''
-      expect(subject).to_not be_valid
-    end
+#     it 'check if the posts_counter numeric' do
+#       subject.posts_counter = ''
+#       expect(subject).to_not be_valid
+#     end
 
-    it 'check if the posts_counter greater than or equal to zero' do
```

Comment:
- [ ] I would recommend removing all the commented code in different files if you are no longer using them.

Reply:
Thank you for your feedback. I'll fix it.